styles/hydrogen.less

```diff
@@ -4,6 +4,7 @@
 // for a full listing of what's available.
 @import "ui-variables";
 @import "syntax-variables";
+@import (less) "../node_modules/react-table/react-table.css";
```

Comment:
Oh really, it would be so annoying.

Do you think is it better to copy the .css file from the original react-table module and paste it in `styles` directory in order to ensure there is the file ??

I first did so but changed to import it since I thought it would more robust to the package update.